src/pages/blog/[...id].astro

```diff
@@ -4,18 +4,17 @@ import BlogLayout from '../../layouts/BlogLayout.astro';
 import CommunityLinks from "../../components/blog/CommunityLinks.astro";
 
 export async function getStaticPaths() {
-  const posts = await getCollection('blog');
-  return posts.map(post => ({
-    params: { id: post.id },
-    props: { post },
+  const blogEntries = await getCollection('blog');
```

Comment:
simply reverted back to the original variable names. realised the change was due to copying and pasting from the documentation.